src/integrationTest/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersControllerIT.java

```diff
@@ -124,6 +126,110 @@ void setUp() {
+            @Override
+            @DisplayName(DISPLAY_ALL_AUTH_OK)
+            public void should2xxSuccessfulForHappyPath() throws Exception {
+                assert2xxSuccessfulForHappyPath(mockMvc, happyPathRequestBuilder, AUTHORISED_CRUD_SERVICE);
+                setUp(); // re-run setup to ensure CPO details are unique for second test call
+                assert2xxSuccessfulForHappyPath(mockMvc, happyPathRequestBuilder, AUTHORISED_CREATE_SERVICE);
+            }
+
+            @Test
+            @Override
+            @DisplayName(DISPLAY_AUTH_MISSING)
+            public void should401ForMissingAuthToken() throws Exception {
+                assert401ForMissingAuthToken(mockMvc, happyPathRequestBuilder, AUTHORISED_CRUD_SERVICE);
+            }
+
+            @Test
+            @Override
+            @DisplayName(DISPLAY_AUTH_MALFORMED)
+            public void should401ForMalformedAuthToken() throws Exception {
+                assert401ForMalformedAuthToken(mockMvc, happyPathRequestBuilder, AUTHORISED_CRUD_SERVICE);
+            }
+
+            @Test
+            @Override
+            @DisplayName(DISPLAY_AUTH_EXPIRED)
+            public void should401ForExpiredAuthToken() throws Exception {
+                assert401ForExpiredAuthToken(mockMvc, happyPathRequestBuilder, AUTHORISED_CRUD_SERVICE);
+            }
+
+            @Test
+            @Override
+            @DisplayName(DISPLAY_S2S_AUTH_MISSING)
+            public void should401ForMissingServiceAuthToken() throws Exception {
+                assert401ForMissingServiceAuthToken(mockMvc, happyPathRequestBuilder, AUTHORISED_CRUD_SERVICE);
+            }
+
+            @Test
+            @Override
+            @DisplayName(DISPLAY_S2S_AUTH_MALFORMED)
+            public void should401ForMalformedServiceAuthToken() throws Exception {
+                assert401ForMalformedServiceAuthToken(mockMvc, happyPathRequestBuilder, AUTHORISED_CRUD_SERVICE);
+            }
+
+            @Test
+            @Override
+            @DisplayName(DISPLAY_S2S_AUTH_EXPIRED)
+            public void should401ForExpiredServiceAuthToken() throws Exception {
+                assert401ForExpiredServiceAuthToken(mockMvc, happyPathRequestBuilder, AUTHORISED_CRUD_SERVICE);
+            }
+
+            @Test
+            @Override
+            @DisplayName(DISPLAY_S2S_AUTH_UNAUTHORISED)
+            public void should403ForUnauthorisedServiceAuthToken() throws Exception {
+                assert403ForUnauthorisedServiceAuthToken(mockMvc, happyPathRequestBuilder);
+            }
+
+            @Test
+            @Override
+            @DisplayName(DISPLAY_S2S_PERMISSION_MISSING)
+            public void should403ForServiceMissingS2sPermission() throws Exception {
+                assert403ForServiceMissingS2sPermission(mockMvc, happyPathRequestBuilder, AUTHORISED_READ_SERVICE);
+                assert403ForServiceMissingS2sPermission(mockMvc, happyPathRequestBuilder, AUTHORISED_UPDATE_SERVICE);
```

Comment:
i guess any single test with the absence of create permission would do.

Reply:
NB: Multiple test calls still present after refactor but implementation cleaner.